Framework/API/test/IFunction1DTest.h

```diff
@@ -61,13 +61,13 @@ class IFunction1DTest : public CxxTest::TestSuite {
     FunctionValues values(domain);
     function.function(domain, values);
     for (size_t i = 0; i < domain.size(); ++i) {
-      TS_ASSERT_EQUALS(values.getCalculated(i), A * (1.0 + 0.1 * double(i)) + B);
+      TS_ASSERT_DELTA(values.getCalculated(i), A * (1.0 + 0.1 * double(i)) + B, 1e-8);
```

Comment:
Noting that changes to this unit test were necessary as a [floating-point precision error](https://builds.mantidproject.org/job/pull_requests-conda-osx/2936/testReport/junit/projectroot.Framework.API/test/APITest_IFunction1DTest/) was introduced due to updates to the versions of clang, clang++ and/or LLVM. This PR brings these versions in line with the `conda-forge` pins. 

I've looked at the patch notes and its unclear exactly what change has caused this.